frontend/src/scenes/settings/SettingsMap.tsx

```diff
@@ -395,6 +396,11 @@ export const SETTINGS_MAP: SettingSection[] = [
         id: 'environment-danger-zone',
         title: 'Danger zone',
         settings: [
+            {
+                id: 'project-move',
+                title: 'Move environment',
+                component: <ProjectMove />,
```

Comment:
Could we feature flag this just so we can test it on prod first before it goes out to everyone?

Reply:
I can... but then I demand an approval as this PR is taking for ever for such an easy feature 😅